packages/core-mobile/app/services/notifications/balanceChange/subscribeForBalanceChange.ts

```diff
@@ -0,0 +1,39 @@
+import Logger from 'utils/Logger'
+import AppCheckService from 'services/fcm/AppCheckService'
+
+export async function subscribeForBalanceChange({
+  deviceArn,
+  chainIds,
+  addresses
+}: {
+  deviceArn: string
+  chainIds: string[]
+  addresses: string[]
+}): Promise<{ message: 'ok' }> {
+  const appCheckToken = await AppCheckService.getToken()
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'X-Firebase-AppCheck': appCheckToken.token
+    },
+    body: JSON.stringify({
+      deviceArn,
+      chainIds,
+      addresses
+    })
+  }
+
+  const response = await fetch(
+    'https://core-notification-sender-api.avax-test.network/v1/push/balance-changes/subscribe',
```

Comment:
let's use env for this

Reply:
[Extract notification api to .env](https://github.com/ava-labs/core-mobile/pull/1659/commits/50f58812ab5c64e9f78a0f82bfb60a662699d772)